x-pack/plugins/apm/server/lib/metrics/query_types.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { StringMap } from 'x-pack/plugins/apm/typings/common';
+import { Setup } from '../helpers/setup_request';
+
+export interface MetricsRequestArgs {
+  serviceName: string;
+  setup: Setup;
+  timeseriesAggregates?: StringMap<any>;
+  otherAggregates?: StringMap<any>;
```

Comment:
I've only heard it referred to as "aggregations". So I think the above two should be `timeseriesAggregations` and `otherAggregates.`
Btw. what do you think of `bucketAggregations` and `totalAggregations` or something like that to distinguish that the first set is run per histogram bucket, the other set is global?